core/src/main/java/org/apache/accumulo/core/iterators/Combiner.java

```diff
@@ -313,4 +392,48 @@ public static void setColumns(IteratorSetting is, List<IteratorSetting.Column> c
   public static void setCombineAllColumns(IteratorSetting is, boolean combineAllColumns) {
     is.addOption(ALL_OPTION, Boolean.toString(combineAllColumns));
   }
+
+  /**
+   * @since 1.6.4 1.7.1 1.8.0
```

Comment:
I like this approach because it simplified things.  In bbea55d I got rid of the enum and swithced to a boolean.  There is still an addition of a public method to Combiner.